nixio/dimension_type.py

```diff
@@ -6,8 +6,9 @@
 # modification, are permitted under the terms of the BSD License. See
 # LICENSE file in the root of the Project.
 
+from enum import Enum
 
-class DimensionType(object):
+class DimensionType(Enum):
     Sample = "sample"
```

Comment:
Case should be maintained. Here `Sample="sample"` i.e in lowercase and in `link_type.py` it is in uppercase.